### PR TITLE
ft: ZENKO-1384: md proxy healthcheck

### PR DESCRIPTION
**Pull request by ploki**

### CODE REVIEW

**Comment by ploki** [comment]
@bert-e reset

File: lib/storage/metadata/proxy/BucketdRoutes.js [hunk 1]
```diff
@@ -106,6 +106,18 @@ class BucketdRoutes {
                 sendResponse(req, res, logger, err, data));
     }
 
+    // `admin' context methods
+
+    _checkHealth(req, res, logger) {
+        return this._metadataWrapper.checkHealth(logger, (err, resp) => {
+            if (err) {
+                logger.error('Failed the health check', { error: err });
```
Comment (nicolas2bert) [inline lib/storage/metadata/proxy/BucketdRoutes.js]: you might want to specify the method name in the error log

```suggestion
                logger.error('Failed the health check', { error: err, method: "_checkHealth" });
```

**Review by nicolas2bert — APPROVED** [review]

**Review by alexanderchan-scality — APPROVED** [review]

**Comment by ploki** [comment]
@bert-e approve

**Comment by ploki** [comment]
@bert-e reset

**Review by rahulreddy — APPROVED** [review]

**Pull request merged**